readinglist/backend/memory.py

```diff
@@ -13,6 +13,7 @@ class Memory(BackendBase):
     def __init__(self, *args, **kwargs):
         super(Memory, self).__init__(*args, **kwargs)
         self._store = tree()
+        self._revisions = defaultdict(int)
```

Comment:
This means the memory backend is not usable in a scalable environment. We should probably tell this in the documentation.